proxy/logging/Log.cc

```diff
@@ -594,6 +594,11 @@ Log::init_fields()
   global_field_list.add(field, false);
   ink_hash_table_insert(field_symbol_hash, "pqhl", field);
 
+  field = new LogField("proxy_req_net_hdr_len", "pqnhl", LogField::sINT, &LogAccess::marshal_proxy_req_net_hdr_len,
+                       &LogAccess::unmarshal_int_to_str);
+  global_field_list.add(field, false);
+  ink_hash_table_insert(field_symbol_hash, "pqnhl", field);
+
```

Comment:
Looking at this documentation and re-reading sudheerv's remarks at 8:26 and the eventual wording of the issue lead me to think that adding a new log tag is not the right fix. If you agree I will commit again removing the new field and modifying "pqhl" to match the documentation.